tests/raftstore/test_lease_read.rs

```diff
@@ -0,0 +1,256 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! A module contains test cases for lease read on Raft leader.
+
+use std::thread;
+use std::time::Duration;
+
+use kvproto::metapb::{Peer, Region};
+use kvproto::raft_cmdpb::CmdType;
+use kvproto::raft_serverpb::RaftLocalState;
+use tikv::raftstore::{Error, Result};
+use tikv::raftstore::store::keys;
+use tikv::raftstore::store::engine::Peekable;
+use tikv::storage;
+use tikv::util::escape;
+
+use super::cluster::{Cluster, Simulator};
+use super::node::new_node_cluster;
+use super::server::new_server_cluster;
+use super::transport_simulate::*;
+use super::util::*;
+
+// Perform a local read on the specified peer.
+fn do_local_read_on_peer<T: Simulator>(cluster: &mut Cluster<T>,
```

Comment:
It's confusing to use the same term `local read` both on client perspective(which means to issue a request without field `read_quorum`) and on server perspective(which means to perform a read on local db without reach the quorum).
Let's change this function name to simple `read_on_peer`.